MAUI/Scheduler/appointments.md

```diff
@@ -824,11 +824,11 @@ N>
 
 N> [View sample in GitHub](https://github.com/SyncfusionExamples/maui-scheduler-examples/tree/main/RecursiveExceptionAppointment/BusinessObject)
```

Comment:
Add an exception appointment to the recurrence pattern - custom app - make it H4